client/dom/ColorScale.ts

```diff
@@ -0,0 +1,254 @@
+import { scaleLinear } from 'd3-scale'
+import { Elem, SvgG } from '../types/d3'
```

Comment:
wrong import. d3.d.ts

Reply:
@xzhou82 `.d.ts` is the extension. We shouldn't need to include the extension on dictionary type files in a typescript file.